vm/llvm-jit/src/compat/compat_llvm.cpp

```diff
@@ -1,10 +1,39 @@
 #include "spdlog/spdlog.h"
 #include <bpftime_vm_compat.hpp>
 #include <cerrno>
+#include <cstdint>
 #include <memory>
 #include <ebpf_inst.h>
 #include "compat_llvm.hpp"
 #include "../llvm/llvm_jit_context.hpp"
+
+static uint64_t default_map_by_fd(uint32_t)
+{
+	SPDLOG_WARN("map_by_fd not defined, but called");
```

Comment:
I don't think we need to warn here, we can make the input val the same as output.